From 9b18c72aa241596ef765905dbc7278c313d88305 Mon Sep 17 00:00:00 2001
From: Michael Andres <ma@suse.de>
Date: Sat, 7 Dec 2024 22:10:34 +0100
Subject: [PATCH] RpmDb: Recognize rpmdb.sqlite as database file (#593)

---
 zypp/target/rpm/librpmDb.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/zypp/target/rpm/librpmDb.cc b/zypp/target/rpm/librpmDb.cc
index a6ea086f9..f5368be46 100644
--- a/zypp/target/rpm/librpmDb.cc
+++ b/zypp/target/rpm/librpmDb.cc
@@ -71,7 +71,8 @@ namespace rpm
     inline bool dbExists( const Pathname & root_r, const Pathname & dbPath_r )
     {
       Pathname dbdir { root_r / sanitizedDbPath( root_r, dbPath_r ) };
-      return PathInfo(dbdir).isDir() && ( PathInfo(dbdir/"Packages").isFile() || PathInfo(dbdir/"Packages.db").isFile() );
+      return PathInfo(dbdir).isDir() &&
+      ( PathInfo(dbdir/"Packages").isFile() || PathInfo(dbdir/"Packages.db").isFile() || PathInfo(dbdir/"rpmdb.sqlite").isFile() );
     }
 
   } // namespace internal
