diff -uraN libzypp-17.31.10/zypp/Arch.cc omv-libzypp-17.31.10/zypp/Arch.cc
--- libzypp-17.31.10/zypp/Arch.cc	2023-04-13 16:49:34.000000000 +0200
+++ omv-libzypp-17.31.10/zypp/Arch.cc	2023-04-16 16:06:08.507689074 +0200
@@ -164,6 +164,7 @@
   DEF_BUILTIN( x86_64_v2 );
   DEF_BUILTIN( x86_64_v3 );
   DEF_BUILTIN( x86_64_v4 );
+  DEF_BUILTIN( znver1 );
 
   DEF_BUILTIN( pentium3 );
   DEF_BUILTIN( pentium4 );
@@ -316,6 +317,7 @@
         defCompatibleWith( a_x86_64_v2(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64() );
         defCompatibleWith( a_x86_64_v3(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2() );
         defCompatibleWith( a_x86_64_v4(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2(),a_x86_64_v3() );
+        defCompatibleWith( a_znver1(),		a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2(),a_x86_64_v3(),a_x86_64_v4() );
 
         defCompatibleWith( a_pentium3(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686() );
         defCompatibleWith( a_pentium4(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_pentium3() );
diff -uraN libzypp-17.31.10/zypp/Arch.h omv-libzypp-17.31.10/zypp/Arch.h
--- libzypp-17.31.10/zypp/Arch.h	2023-04-13 16:49:34.000000000 +0200
+++ omv-libzypp-17.31.10/zypp/Arch.h	2023-04-16 17:02:16.372029672 +0200
@@ -178,6 +178,8 @@
   /** \relates Arch */
   extern const Arch Arch_x86_64;
   /** \relates Arch */
+  extern const Arch Arch_znver1;
+  /** \relates Arch */
   extern const Arch Arch_athlon;
   /** \relates Arch */
   extern const Arch Arch_i686;
diff -uraN libzypp-17.31.10/zypp/parser/yum/schema/common-inc.rnc omv-libzypp-17.31.10/zypp/parser/yum/schema/common-inc.rnc
--- libzypp-17.31.10/zypp/parser/yum/schema/common-inc.rnc	2023-04-13 16:49:34.000000000 +0200
+++ omv-libzypp-17.31.10/zypp/parser/yum/schema/common-inc.rnc	2023-04-16 17:04:17.017555333 +0200
@@ -34,6 +34,7 @@
             | "x86_64"
             | "pentium3"
             | "pentium4"
+            | "znver1"
             | "s390"
             | "s390x"
             | "ppc"
