diff -up libzypp-17.35.8/zypp-core/CMakeLists.txt.1~ libzypp-17.35.8/zypp-core/CMakeLists.txt
--- libzypp-17.35.8/zypp-core/CMakeLists.txt.1~	2024-08-04 12:28:13.856315622 +0200
+++ libzypp-17.35.8/zypp-core/CMakeLists.txt	2024-08-04 12:28:50.808716953 +0200
@@ -440,6 +440,8 @@ target_link_libraries( zypp-core INTERFA
 TARGET_LINK_LIBRARIES( zypp-core INTERFACE ${ZLIB_LIBRARY} )
 TARGET_LINK_LIBRARIES( zypp-core INTERFACE ${UTIL_LIBRARY} )
 
+TARGET_LINK_LIBRARIES( zypp-core PRIVATE absl_hash absl_log_internal_message absl_log_internal_check_op)
+
 IF (ENABLE_ZSTD_COMPRESSION)
   TARGET_LINK_LIBRARIES( zypp-core INTERFACE ${ZSTD_LIBRARY})
 ENDIF (ENABLE_ZSTD_COMPRESSION)
