diff -up libzypp-17.31.23/zypp/CMakeLists.txt.4~ libzypp-17.31.23/zypp/CMakeLists.txt
--- libzypp-17.31.23/zypp/CMakeLists.txt.4~	2023-11-03 01:40:05.146613857 +0100
+++ libzypp-17.31.23/zypp/CMakeLists.txt	2023-11-03 01:41:39.042313243 +0100
@@ -912,7 +912,7 @@ macro( ADDZYPPLIB LIBNAME )
   #TARGET_LINK_LIBRARIES(${LIBNAME} ${SIGNALS_LIBRARY})
   TARGET_LINK_LIBRARIES(${LIBNAME} ${Boost_THREAD_LIBRARY})
   TARGET_LINK_LIBRARIES(${LIBNAME} ${GPGME_LIBRARIES})
-  target_link_libraries(${LIBNAME} ${YAML_CPP_LIBRARIES})
+  target_link_libraries(${LIBNAME} ${YAML_CPP_LIBRARIES} -lyaml-cpp)
   TARGET_LINK_LIBRARIES(${LIBNAME} pthread )
 
   IF ( UDEV_FOUND )
