diff -up libzypp-17.38.7/zyppng/lib/CMakeLists.txt.5~ libzypp-17.38.7/zyppng/lib/CMakeLists.txt
--- libzypp-17.38.7/zyppng/lib/CMakeLists.txt.5~	2026-04-27 16:49:30.000000000 +0200
+++ libzypp-17.38.7/zyppng/lib/CMakeLists.txt	2026-05-06 22:50:44.906737757 +0200
@@ -23,6 +23,8 @@ ADD_LIBRARY( ${target} STATIC zypp/dummy
 SET_TARGET_PROPERTIES ( ${target} PROPERTIES VERSION "${LIBZYPP_VERSION_INFO}" )
 SET_TARGET_PROPERTIES ( ${target} PROPERTIES SOVERSION "${LIBZYPP_SOVERSION_INFO}" )
 
+target_link_libraries( ${target} PUBLIC zypp)
+
 target_link_libraries( ${target} PRIVATE zyppng_lib_compiler_flags )
 target_link_libraries( ${target} PRIVATE zyppng-objlib )
 target_link_libraries( ${target} PRIVATE zyppng-commit-proto-obj )
