--- libzypp-14.30.0/zypp/UserData.h.orig	2014-10-19 08:37:41.546722812 +0100
+++ libzypp-14.30.0/zypp/UserData.h	2014-10-19 08:37:54.017721893 +0100
@@ -60,8 +60,8 @@
     };
 
     /** \relates UserData Stream output */
-    inline std::ostream & operator<<( std::ostream & str, const UserData & obj )
-    { return str << "UserData(" << obj.type() << ")";}
+    //inline std::ostream & operator<<( std::ostream & str, const UserData & obj )
+    //{ return str << "UserData(" << obj.type() << ")";}
 
   } // namespace callback
   ///////////////////////////////////////////////////////////////////
