diff -up libzypp-17.31.11/zypp/Arch.cc.3~ libzypp-17.31.11/zypp/Arch.cc
--- libzypp-17.31.11/zypp/Arch.cc.3~	2023-05-08 16:03:20.483373701 +0200
+++ libzypp-17.31.11/zypp/Arch.cc	2023-05-08 16:08:01.328533544 +0200
@@ -164,6 +164,7 @@ namespace zypp
   DEF_BUILTIN( x86_64_v2 );
   DEF_BUILTIN( x86_64_v3 );
   DEF_BUILTIN( x86_64_v4 );
+  DEF_BUILTIN( znver1 );
 
   DEF_BUILTIN( pentium3 );
   DEF_BUILTIN( pentium4 );
@@ -316,6 +317,7 @@ namespace zypp
         defCompatibleWith( a_x86_64_v2(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64() );
         defCompatibleWith( a_x86_64_v3(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2() );
         defCompatibleWith( a_x86_64_v4(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2(),a_x86_64_v3() );
+	defCompatibleWith( a_znver1(),		a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64(),a_x86_64_v2(),a_x86_64_v3() );
 
         defCompatibleWith( a_pentium3(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686() );
         defCompatibleWith( a_pentium4(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_pentium3() );
diff -up libzypp-17.31.11/zypp/Arch.h.3~ libzypp-17.31.11/zypp/Arch.h
--- libzypp-17.31.11/zypp/Arch.h.3~	2023-05-08 16:03:20.483373701 +0200
+++ libzypp-17.31.11/zypp/Arch.h	2023-05-08 16:06:07.705658184 +0200
@@ -170,6 +170,8 @@ namespace zypp
   extern const Arch Arch_pentium3;
 
   /** \relates Arch */
+  extern const Arch Arch_znver1;
+  /** \relates Arch */
   extern const Arch Arch_x86_64_v4;
   /** \relates Arch */
   extern const Arch Arch_x86_64_v3;
diff -up libzypp-17.31.11/zypp/parser/yum/schema/common-inc.rng.3~ libzypp-17.31.11/zypp/parser/yum/schema/common-inc.rng
--- libzypp-17.31.11/zypp/parser/yum/schema/common-inc.rng.3~	2023-04-28 13:40:38.000000000 +0200
+++ libzypp-17.31.11/zypp/parser/yum/schema/common-inc.rng	2023-05-08 16:03:20.483373701 +0200
@@ -97,6 +97,7 @@
       <value>x86_64</value>
       <value>pentium3</value>
       <value>pentium4</value>
+      <value>znver1</value>
       <value>s390</value>
       <value>s390x</value>
       <value>ppc</value>
