diff -uraN ./libzypp-17.30.3/zypp/Arch.cc ./omv-libzypp-17.30.3/zypp/Arch.cc
--- ./libzypp-17.30.3/zypp/Arch.cc	2022-07-05 10:29:49.000000000 +0200
+++ ./omv-libzypp-17.30.3/zypp/Arch.cc	2022-07-08 20:11:55.967932510 +0200
@@ -161,6 +161,7 @@
   DEF_BUILTIN( i686 );
   DEF_BUILTIN( athlon );
   DEF_BUILTIN( x86_64 );
+  DEF_BUILTIN( znver1 );
 
   DEF_BUILTIN( pentium3 );
   DEF_BUILTIN( pentium4 );
@@ -192,15 +193,18 @@
 
   DEF_BUILTIN( aarch64 );
 
+  DEF_BUILTIN( armv8hl );
+  DEF_BUILTIN( armv8hcnl );
+  DEF_BUILTIN( armv8hnl );
+  DEF_BUILTIN( armv8l );
+
   DEF_BUILTIN( armv7tnhl );	/* exists? */
   DEF_BUILTIN( armv7thl );	/* exists? */
 
   DEF_BUILTIN( armv7hnl );	/* legacy: */DEF_BUILTIN( armv7nhl );
-  DEF_BUILTIN( armv8hl );
   DEF_BUILTIN( armv7hl );
   DEF_BUILTIN( armv6hl );
 
-  DEF_BUILTIN( armv8l );
   DEF_BUILTIN( armv7l );
   DEF_BUILTIN( armv6l );
   DEF_BUILTIN( armv5tejl );
@@ -310,6 +314,7 @@
         defCompatibleWith( a_i686(),		a_noarch(),a_i386(),a_i486(),a_i586() );
         defCompatibleWith( a_athlon(),		a_noarch(),a_i386(),a_i486(),a_i586(),a_i686() );
         defCompatibleWith( a_x86_64(),		a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon() );
+        defCompatibleWith( a_znver1(),		a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_athlon(),a_x86_64() );
 
         defCompatibleWith( a_pentium3(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686() );
         defCompatibleWith( a_pentium4(),	a_noarch(),a_i386(),a_i486(),a_i586(),a_i686(),a_pentium3() );
@@ -361,6 +366,10 @@
         /*?*/defCompatibleWith( a_armv7thl(),	a_noarch(),a_armv7hl() );
         /*?*/defCompatibleWith( a_armv7tnhl(),	a_noarch(),a_armv7hl(),a_armv7nhl(),a_armv7thl() );
 
+        defCompatibleWith( a_armv8hl(),		a_noarch(),a_armv7hl() );
+	    defCompatibleWith( a_armv8hnl(),	a_noarch(),a_armv7hl(),a_armv7hnl(),a_armv8hl() );
+        defCompatibleWith( a_armv8hcnl(),	a_noarch(),a_armv7hl(),a_armv7hnl(),a_armv8hl(),a_armv8hnl() );
+
         defCompatibleWith( a_aarch64(),		a_noarch() );
         //
         defCompatibleWith( a_riscv64(),		a_noarch() );
diff -uraN ./libzypp-17.30.3/zypp/Arch.h ./omv-libzypp-17.30.3/zypp/Arch.h
--- ./libzypp-17.30.3/zypp/Arch.h	2022-07-05 10:29:49.000000000 +0200
+++ ./omv-libzypp-17.30.3/zypp/Arch.h	2022-07-08 20:14:48.613107799 +0200
@@ -170,6 +170,8 @@
   extern const Arch Arch_pentium3;
 
   /** \relates Arch */
+  extern const Arch Arch_znver1;
+  /** \relates Arch */
   extern const Arch Arch_x86_64;
   /** \relates Arch */
   extern const Arch Arch_athlon;
@@ -242,6 +244,10 @@
   extern const Arch Arch_armv6hl;
 
   /** \relates Arch */
+  extern const Arch Arch_armv8hcnl;
+  /** \relates Arch */
+  extern const Arch Arch_armv8hnl;
+  /** \relates Arch */
   extern const Arch Arch_armv8hl;
   /** \relates Arch */
   extern const Arch Arch_armv8l;
diff -uraN ./libzypp-17.30.3/zypp/parser/yum/schema/common-inc.rnc ./omv-libzypp-17.30.3/zypp/parser/yum/schema/common-inc.rnc
--- ./libzypp-17.30.3/zypp/parser/yum/schema/common-inc.rnc	2022-07-05 10:29:49.000000000 +0200
+++ ./omv-libzypp-17.30.3/zypp/parser/yum/schema/common-inc.rnc	2022-07-08 20:17:12.367947124 +0200
@@ -32,6 +32,7 @@
             | "i686"
             | "athlon"
             | "x86_64"
+            | "znver1"
             | "pentium3"
             | "pentium4"
             | "s390"
@@ -52,6 +53,8 @@
             | "sparcv8"
             | "sparc"
             | "aarch64"
+            | "armv8cnl"
+            | "armv8hnl"
             | "armv8hl"
             | "armv8l"
             | "armv7tnhl"
